apiserver/facades/controller/remoterelations/mock_test.go

```diff
@@ -208,6 +208,14 @@ func (st *mockState) ApplyOperation(op state.ModelOperation) error {
 	return st.NextErr()
 }
 
+func (st *mockState) UpdateControllerForModel(controller crossmodel.ControllerInfo, modelUUID string) error {
```

Comment:
I would loved to have converted all of this to use generated mocks, but we really have to get a move on with this functionality, so I bailed on that aspect.